Added machine learning resource (#4091)
sindresorhus/awesome: readme.md

- [YouTubers](https://github.com/JoseDeFreitas/awesome-youtubers#readme) - Watch video tutorials from YouTubers that teach you about technology.
- [Hackathon](https://github.com/dribdat/awesome-hackathon#readme) - Running fun and productive hackathons.
- [Certificates](https://github.com/PanXProject/awesome-certificates#readme) - Free computer science certifications to showcase your knowledge.

> - [Machine Learning Books](https://github.com/dair-ai/ML-YouTube-Courses#readme) - A curated list of machine learning courses and educational content for beginners to advanced learners.

Reviewer: Remove Access

> - [Machine Learning Books](https://github.com/dair-ai/ML-YouTube-Courses#readme) - A curated list of machine learning courses and educational content for beginners to advanced learners.

Reviewer: > The new entry is internally inconsistent and likely violates the Awesome list submission guidelines: (1) the link points to “ML-YouTube-Courses”, which appears to be a YouTube courses resource repo (not an `awesome-…` list), while the entry title says “Machine Learning Books”; (2) the description starts with “A curated list of…”, which describes the list itself rather than the project/theme. Please either link to an actual Awesome list that matches the title, or update the title/URL and rewrite the description to objectively describe the topic (and place it in the most relevant section, e.g., Books vs Learn).
> ```suggestion
> - [Machine Learning Courses](https://github.com/dair-ai/ML-YouTube-Courses#readme) - Machine learning courses and educational videos for beginners to advanced learners.
> ```

Remove Access
